docs/pages/index.mdx

```diff
@@ -4,40 +4,61 @@ description: Build zero-friction user onboarding and transactions end-to-end wit
 slug: wallets
 ---
 
-<img src="/images/wallets/overviewHeader.png" alt="smart wallets overview" />
+<style>{`
+.fern-card {
+  font-size: 14px;
+}
 
-<div
-  style={{
-    display: "flex",
-    justifyContent: "center",
-    marginBottom: "2rem",
-    gap: "1.5rem",
-  }}
->
-  <Button
+/* Responsive hero images */
+.hero-image-desktop {
+  display: block;
+  width: 100%;
+  height: auto;
+  max-width: 1010px;
+  margin: 0 auto;
+}
+
+.hero-image-mobile {
+  display: none;
+  width: 100%;
```

Comment:
I don't recommend width: 100% here. since it's a vertical image, scaling it horizontally makes it huge. Also gets very pixelated. So I'd also use something higher res
<img width="635" height="924" alt="image" src="https://github.com/user-attachments/assets/4a0a29fb-af22-49be-bad0-3e62ea3345fb" />